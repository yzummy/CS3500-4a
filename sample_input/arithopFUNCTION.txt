{
    fuzzy_socks = function() { -2.71 };
    clock = TRUE & fuzzy_socks
}
