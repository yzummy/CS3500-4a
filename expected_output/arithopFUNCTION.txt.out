Line 4: Arg 2 must be integer or float or bool
